dialogue-clients/src/main/java/com/palantir/dialogue/clients/DialogueClients.java

```diff
@@ -65,19 +66,25 @@ public static ReloadingFactory create(Refreshable<ServicesConfigBlock> scb) {
         Channel getChannel(String serviceName);
     }
 
-    public interface ReloadingStickyFactory {
-        StickyChannels getStickyChannels(String serviceName);
+    public interface ToPerHostClientFactory {
```

Comment:
Name is odd, but should only be used fluently. Maybe `ToPerHostClientFactory` -> `PinnableClientFactory` an `PerHostClientFactory` -> `PinningClientFactory`? Naming is difficult.

Reply:
yeah I got so fed up with trying to come up with a good name for this interface I threw it away completely. The two methods are now just tacked onto the ReloadingClientFactory